pkg/util/validation/validation.go

```diff
@@ -109,3 +109,20 @@ func EmptyOverrides(overriders policyv1alpha1.Overriders) bool {
 	}
 	return true
 }
+
+// ValidateReplicaScheduling validate replicaScheduling and replicaDivisionPreference.
+func ValidateReplicaScheduling(replicaScheduling *policyv1alpha1.ReplicaSchedulingStrategy) error {
+	if replicaScheduling == nil {
+		return nil
+	}
+
+	if replicaScheduling.ReplicaSchedulingType == "" {
```

Comment:
What if make this field in API as `+required`?

Reply:
Maybe this field needs to be set to a default value.

Ask for @RainbowMango to take a look.
/cc @RainbowMango

Reply:
> Maybe this field needs to be set to a default value.

https://github.com/karmada-io/karmada/blob/master/pkg/scheduler/core/generic_scheduler.go#L182-L197

Reply:
What do u think?
@RainbowMango

Reply:
I tend to take `Weighted` as the default value of `replicaDivisionPreference`.

> What if make this field in API as +required?

That's an option, but it is kind of a breaking change. We can consider doing that when promoting to v1alpha2 or beta.

Reply:
For the `replicaSchedulingType`, it should be a `required` to make a clear statement, but for compatibility concerns, I still prefer to take a default value `Divided`.

Reply:
@Garrybest @chaunceyjiang How do you think?

If it is ok to you, I think we can do that by kubebuilder instructions.
```
+kubebuilder:default
```

https://book.kubebuilder.io/reference/markers/crd-validation.html

Reply:
> For the replicaSchedulingType, it should be a required to make a clear statement

Agree.

> I still prefer to take a default value Divided.

I do not understand.
If PP does not fill in `replicaScheduling`, the default behavior of the scheduler is now similar to `Duplicated`.

https://github.com/karmada-io/karmada/blob/master/pkg/scheduler/core/generic_scheduler.go#L182-L197

Reply:
Yes, if `replicaScheduling == null`, the behavior is `Duplicated`.
But if `replicaScheduling  != null`, the behavior is `Divided`, by default.

Reply:
Get it.